cmd/deploy/deploy.go

```diff
@@ -656,6 +677,37 @@ func (dc *DeployCommand) cleanUp(ctx context.Context) {
 	}
 }
 
+func (dc *DeployCommand) createTempOktetoEnvFile() (afero.File, error) {
+	oktetoEnvFileDir, err := afero.TempDir(dc.Fs, "", "")
+	if err != nil {
+		return nil, err
+	}
+
+	oktetoEnvFile, err := dc.Fs.Create(fmt.Sprintf("%s/.env", oktetoEnvFileDir))
+	if err != nil {
+		return nil, err
+	}
+
+	os.Setenv(constants.OktetoEnvFile, oktetoEnvFile.Name())
+	return oktetoEnvFile, nil
+}
+
+func (*DeployCommand) updateEnvironWithOktetoEnvFile(oktetoEnvFile afero.File) error {
+	envMap, err := godotenv.ParseWithLookup(oktetoEnvFile, os.LookupEnv)
```

Comment:
What happen if the env file has one of the lines bad formatted? Do we load all others? Does it fail? 

Since it's something that is the user the one that adds lines to it I think we should just skip that line

Reply:
With the current implementation and following the acceptance criteria, if one of the lines has an unaccepted format we fail the whole deployment. I think that if one of the lines does not have the right format we should fail since that line is part of the dev environment deployment process and it could not be executed correctly so it can leave an undesired state to the development environment